cmd/mtail/main.go

```diff
@@ -227,6 +235,22 @@ func main() {
 		os.Exit(1) //nolint:gocritic // false positive
 	}
 	if *oneShot {
+		if *prometheusOutput {
+			e, err := exporter.New(ctx, nil, store)
+			if err != nil {
+				glog.Error(err)
+				cancel()
+				os.Exit(1) //nolint:gocritic // false positive
+			}
+			err = e.Write(os.Stdout)
+			if err != nil {
+				glog.Error(err)
+				cancel()
+				os.Exit(1) //nolint:gocritic // false positive
+			}
+			cancel()
+			os.Exit(0) //nolint:gocritic // false positive
+		}
```

Comment:
Should this have an `} else {` to not print the json output when using `prometheusOutput`?

Reply:
:+1: